src/ducks/sharing/index.js

```diff
@@ -38,6 +38,44 @@ export const createPermSet = (id, doctype) =>
     }
   })
 
+export const share = async (id, doctype, email, url) => {
+  const target = await cozy.client.fetchJSON('POST', '/sharings/recipient', {
+    email,
+    url
+  })
+
```

Comment:
Maybe not for this PR, but if I recall correctly the plan was to use a dummy implementation of `io.cozy.contacts` here.